csharp/src/Drivers/Apache/Spark/README.md

```diff
@@ -66,7 +69,34 @@ The following table depicts how the Spark ADBC driver converts a Spark type to a
 | USER_DEFINED         | String     | string |
 | VARCHAR              | String     | string |
 
-\* Complex types are returned as strings<br>
+### Apache Spark over HTTP
+
+| Spark Type           | Arrow Type | C# Type |
+| :---                 | :---:      | :---:   |
+| ARRAY*               | String     | string  |
+| BIGINT               | Int64      | long |
+| BINARY               | Binary     | byte[] |
+| BOOLEAN              | Boolean    | bool |
+| CHAR                 | String     | string |
+| DATE*                | *String*   | *string* |
+| DECIMAL*             | *String*   | *string* |
```

Comment:
@CurtHagenlocher 
This is a "Thrift" limitation - if we don't do any further conversion. We have the opportunity to convert the string types to native type - at the expense of memory/performance. Should we do always do the conversion on the client driver? Or possibly allow an option to decide?

Reply:
I think the correct thing is to do the conversion locally, yes. I think it would be very surprising for a user to run a query that produces a decimal or date value and to get that back as a string. I understand that we've been avoiding this for structured types where it's perhaps a little more justifiable but this just seems wrong.

One thing we could do to unblock this checkin is to add an option for converting scalar types and to fail if the value is not explicitly passed as false. This would let us add support in a subsequent change without breaking consumers.

Reply:
Introduced option `data_type_conv` with value `none` to indicate that no conversion should be done. Indicated future support for `scalar` to convert date, decimal and timestamp types.